hack/e2e.sh

```diff
@@ -36,7 +36,9 @@ oc new-app -f template/cincinnati.yaml \
   -p GB_CINCINNATI_REPO="redhat/openshift-cincinnati-test-public-manual" \
   -p GB_CPU_REQUEST=50m \
   -p PE_CPU_REQUEST=50m \
-  -p RUST_BACKTRACE="1"
+  -p RUST_BACKTRACE="1" \
+  -p GB_REGISTRY_CREDENTIALS_PATH="" \
+  ;
```

Comment:
This would ignore the exit code of `oc new-app`, isn't it? Lets remove that

Reply:
No, it's a trick which allows having a `\` behind each semantically meaningful line. The `;` explicitly terminates the command which is split across several lines. It doesn't change the behavior to the best of my knowledge. You may be confusing it for the construct
```
some_command || \
    :
```

Reply:
Oh, pretty cool